Cargo.toml

```diff
@@ -1,21 +1,16 @@
 [workspace]
-resolver = "2"
+resolver = "3"
 members = [
     "aead",
     "async-signature",
     "cipher",
     "crypto",
     "crypto-common",
     "digest",
-    "elliptic-curve",
     "kem",
     "password-hash",
     "signature",
     "signature_derive",
     "universal-hash",
 ]
-
-[patch.crates-io]
-crypto-common = { path = "./crypto-common" }
-digest = { path = "./digest" }
-signature = { path = "./signature" }
+exclude = ["elliptic-curve"]
```

Comment:
`elliptic-curve` is temporarily excluded from the workspace because it can not be migrated to `hybrid-array` v0.3 yet, but it pulls `block-buffer` v0.11.0-rc.4 in tests, which depends on hybrid-array v0.2. It causes breakage, so we have to manually downgrade `block-buffer` to v0.11.0-rc.3.